lib/style.js

```diff
@@ -85,7 +85,9 @@ const parseStylesheet = (css, dynamic) => {
     if (cssNode.type === 'Atrule') {
       if (
         cssNode.name === 'keyframes' ||
-        cssNode.name === '-webkit-keyframes'
+        cssNode.name === '-webkit-keyframes' ||
+        cssNode.name === '-o-keyframes' ||
+        cssNode.name === '-moz-keyframes'
```

Comment:
Wouldn't it be clearer to use `includes` and an array here?

Reply:
yeah of course